src/modules/github/action.ts

```diff
@@ -5,18 +5,19 @@ export const SEARCH_GITHUB_REPOSITORIES_REQUEST = 'SEARCH_GITHUB_REPOSITORIES_RE
 export const SEARCH_GITHUB_REPOSITORIES_SUCCESS = 'SEARCH_GITHUB_REPOSITORIES_SUCCESS';
 export const SEARCH_GITHUB_REPOSITORIES_ERROR = 'SEARCH_GITHUB_REPOSITORIES_ERROR';
 
-export const searchGithubRepositories = (name: string) => async (dispatch: Dispatch) => {
+export const searchGithubRepositories = (name: string, currentPage = 1, perPage: number) => async (dispatch: Dispatch) => {
```

Comment:
Always put optional arguments at the end of argument list because their default value is `undefined`:
```js
export const searchGithubRepositories = (name: string, perPage: number, currentPage = 1) => {...}
```